tests/end-to-end/external-solution-integration/internal/scenario/connectivity_adapter/scenario.go

```diff
@@ -12,13 +14,15 @@ type Scenario struct {
 	domain        string
 	testID        string
 	skipSSLVerify bool
+	waitTime      time.Duration
 }
 
 // AddFlags adds CLI flags to given FlagSet
 func (s *Scenario) AddFlags(set *pflag.FlagSet) {
 	pflag.StringVar(&s.domain, "domain", "kyma.local", "domain")
 	pflag.StringVar(&s.testID, "testID", "connectivity-adapter-e2e", "domain")
 	pflag.BoolVar(&s.skipSSLVerify, "skipSSLVerify", false, "Skip verification of service SSL certificates")
+	pflag.DurationVar(&s.waitTime, "waitTime", time.Duration(10)*time.Second, "Wait time in seconds, e.g. 5s")
```

Comment:
Same for the flags I feel like `waitTime` is to general term and it is only used before sending the event. I would suggest renaming it to something more descriptive like `eventSendDelay`.

Reply:
I agree, also the flag description can be changed: `Wait time in seconds, e.g. 5s`.  You can provide any duration there `1s, 1m, 1h`.

Reply:
Cause it's a ```time.Duration``` you can configure it to have any "go" duration, of course. I wanted to constrain it to seconds, otherwise it could be a sign for a true error in Event Mesh.

Reply:
Make comment more explicit.